src/workloads/c2c/ChangeEventApplication.yml

```diff
@@ -55,7 +57,13 @@ Actors:
   - LoadConfig:
       Path: *scriptsPath
       Key: PollForCEA
+  - LoadConfig:
```

Comment:
In this way, we pre-buffer the change events before CEA starts to overcome the genny slowness issue we observed before.